### ISSUE TITLE
report of continued issues with closed report #2418

### ISSUE THREAD

**Issue by hothedhunter** [dalaranwow/dalaran-wow#5583]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. jousting argent tournament
2. playing the blood elf toon
3. not getting marks after credited the win
as reported in issue #2418, i notice that many of these issues are still remaining. while jousting at the argent tournament, you can be credited the win, however you do not get the marks for the win. some NPC's will glitch and not complete the fight at 500 health remaining, some will run out of the joust area and health will reset. toon health bar shows full health with all 3 stacked shields however you become dismounted and killed. this occurs with the jousting and the champion jousting. many times have to repeat the joust many many times without receiving the marks for the win.
**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by alexuktc** [comment]
This happens for all races. Despite this I have the Crusader title on 2 toons on Algalon. 

Tips on how to make it much less likely not to get credited:
- use the Kodo mount. I have found it to be the most reliable, with the Argent Charger being the worst followed by the org wolf.
- Finish with the melee attack, not the charge.

**Comment by hothedhunter** [comment]
but you cannot complete the daily quest unless you are on the mount for the race you are working on. so when you are working on orgrimmar, you have to use the wolf, silvermoon, you have to use the bird....and so on. so what are you supposed to do for the different ones

**Comment by hothedhunter** [comment]
ok and we are on the first set of dailies

Sent from Yahoo Mail on Android<https://go.onelink.me/107872968?pid=InProduct&c=Global_Internal_YGrowth_AndroidEmailSig__AndroidUsers&af_wl=ym&af_sub1=Internal&af_sub2=Global_YGrowth&af_sub3=EmailSignature>

On Tue, Aug 14, 2018 at 3:20 AM, alexuktc
<notifications@github.com> wrote:

Not true. This only applies for the first set of dailies. After that you choose which city to go for.

**Comment by galgnu** [comment]
One reason for not getting marks is if killing blow are done with charge. If you don' t do that you will get marks+win 99/100 times. 
There are minor glichtes but I got my crusader title without any real issues.
